src/pages/projects/AddProjectManagerForm.tsx

```diff
@@ -56,11 +89,20 @@ export const AddProjectManagerForm = ({ toggleModal, suggestedProjectManager }:
         setSelectedPerson={setSelectedPerson}
         searchTerm={searchTerm}
         setSearchTerm={setSearchTerm}
-        selectAffiliations={SelectAffiliations.SINGLE}
+        selectAffiliations={hasAffiliation ? SelectAffiliations.NO_SELECT : SelectAffiliations.SINGLE}
       />
-      <StyledRightAlignedFooter>
+      <StyledRightAlignedFooter sx={{ mt: '2rem' }}>
+        <Box sx={{ flexGrow: '1' }}>
+          <Button
```

Comment:
samme som i `AddProjectContributorForm`
```suggestion
          <Button
            sx={{ mr: 'auto' }}
```